glean-core/python/glean/glean.py

```diff
@@ -55,11 +57,18 @@ class Glean:
     # and saved between test runs.
     _ping_type_queue: Set["PingType"] = set()
 
+    # The application id to send in the ping.
+    _application_id: str = "glean-python"
+
+    # The version of the application sending Glean data.
+    _application_version: Optional[str] = None
+
     @classmethod
     def initialize(
         cls,
         configuration: Optional[Configuration] = None,
         application_id: Optional[str] = None,
+        application_version: Optional[str] = None,
```

Comment:
Can we move this and the above in the configuration object? This is where the `packageName` (from which the app id is derived) lives in [Kotlin](https://github.com/mozilla/glean/blob/e27fc8abe79741da7a45d0602babf8c334db1372/glean-core/android/src/main/java/mozilla/telemetry/glean/config/Configuration.kt#L23). We should also remove the user agent property from there (sorry, I missed that during my review) since we don't allow customizing it in the Glean SDK.

Reply:
Well, looking closer, I don't think that `Configuration.packageName` is used anywhere in Kotlin.  Filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1595487) about that.

Since the Kotlin version pulls this "out of thin air" from the `applicationContext` object, and the same is true of `data_dir`, I thought it was more consistent with Kotlin to have this be part of the `Glean` object.  My thinking is "the Configuration object should be pretty consistent across platforms" and the Glean init params are for platform-specific things.  But I guess I don't feel too strongly about it.  Alternatively, we could put everything on the configuration object (but then we should probably also move `data_dir` to the configuration object, too).

Reply:
Fair point and good catch. Let's leave this as it is for now.